default.cfg.example

```diff
@@ -36,8 +36,20 @@ gaptop = 200
 
 #Daily lend rate threshold after which we offer lends for x days as opposed to 2.
 #If set to 0 all offers will be placed for a 2 day period (0.003-5)
-xdaythreshold = 0.2
-xdays = 60
+vdaythreshold = 0.4
+vdays = 60
+
+wdaythreshold = 0.35
+wdays = 45
+
+xdaythreshold = 0.32
+xdays = 30
+
+ydaythreshold = 0.31
+ydays = 15
+
+zdaythreshold = 0.3
+zdays = 4
```

Comment:
Sorry for the lack of reply... been preoccupied with other stuff recently. I'm 100% ok with any changes.